src/ui.css

```
@@ -86,45 +87,52 @@ body {
   color: #00ccd1;
   margin-bottom: 20px;
 }
-select,
+
 input {
   padding: 2px 0;
   font-size: 16px;
   width: 100%;
   border: none;
   outline: none;
+  margin-right: 15px;
 }
-.searchInput {
+
+.searchBar {
```

```suggestion
.search-bar {
```
let's try to follow the same convention

---

okay